BlogAPI.Persistence/EntityFrameworkCore/Configurations/BlogPostConfiguration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using BlogAPI.Persistence.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace BlogAPI.Persistence.Configurations
+{
+    internal class BlogPostConfiguration : IEntityTypeConfiguration<BlogPost>
+    {
+        public void Configure(EntityTypeBuilder<BlogPost> builder)
+        {
+            builder.HasKey(c => c.Id);
+            builder.Property(c => c.Id).ValueGeneratedOnAdd();
+            builder.Property(c =>  c.Title)
```

Comment:
`c` isn't very definitive, perhaps it would make sense to use the full name, because a lot of things can be `c`.